src/Controller/PagesController.php

```diff
@@ -31,14 +31,13 @@ class PagesController extends AppController
     /**
      * Displays a view
      *
+     * @param array ...$path
```

Comment:
I am not sure if `array` is the right type to use. Variadic args will always be an array so it would be more useful to hint the type of individual items in the array, `string` in this case. We could refer to what others do.

e.g. https://philsturgeon.uk/php/2013/08/30/potential-variadic-function-syntax-for-php-56/#pro-type-hinting